### ISSUE TITLE
implement reconstruct 

### ISSUE THREAD

**Issue by xiang90** [c-fs/cfs#28]
use https://github.com/tsuraan/Jerasure/blob/master/src/jerasure.c#L1453.

**Comment by tnachen** [comment]
@xiang90 looks like we need to hook up the server to initiate request to other servers for reconstruction right? looks like  you've already hook up jerasure, perhaps I can help doing the calls part

**Comment by xiang90** [comment]
@tnachen Right. Basically we want to pull source files from other cfs nodes to /dev/shm (tmpfs), then reconstruct the target files onto disk.

**Comment by tnachen** [comment]
@xiang90 can you add me to the org? I can't assign myself this

**Comment by yichengq** [comment]
@tnachen We start to roll up the wheel again. Are you still working on this one?

**Comment by tnachen** [comment]
@yichengq hey good to know! I have a WIP patch in my repo that I can try to finish this few days. I forgot about this a bit and did other stuff, I have time to work on this now so I can look at it.

**Comment by yichengq** [comment]
@tnachen :+1:

**Comment by yichengq** [comment]
@tnachen How is it going? I could work it with you if you meet problems with it. Or I can continue with your WIP patch too.

**Comment by tnachen** [comment]
@yichengq sorry got occupied again earlier :( I do have some time but I think it's safer to give it to you if you're more available, or we can even work together. Yes please take over from my WIP patch
